docs/gmi.md

```diff
@@ -233,7 +233,7 @@ gmi.account.authorise(sessionId)
 
 ### Achievements
 
-There are several functions in the GMI to support in-game achievements, however these functions are currently disabled and can be ignored until the achievement system is finalised and enabled.
+This is an optional feature that you can include in your game. See the [achievements documentation](achievements.md) for more information.
```

Comment:
May not be optional. Phrase it similar to how we've got centralised settings instead.

Reply:
Sure, though one of the ACs on the ticket said to say it was optional